.gitignore

```diff
@@ -6,4 +6,6 @@ build/
 .pytest_cache/
 .env
 .idea/
+.coverage
+.vscode/
```

Comment:
Not sure why you added the vscode folder to gitignore? It can actually be handy to have this submitted to github so that we can all share the same vscode setup when programming in this repo

Reply:
Forgot to push this comment.
.idea is the same thing (intellij)
Why not ignore for now until use?

Reply:
Here you go @jamiehewitt15 => https://github.com/oceanprotocol/team-sailfish/issues/41
Reverting this change, resolving the conversation, and pushing rest of PR.